README.md

```diff
@@ -0,0 +1,53 @@
+<img src="logo.png" align="right" />
+
+# UCheck
+<!-- Banges -->
+[![releases](https://img.shields.io/github/v/release/DLochmelis33/UCheck.svg)](https://github.com/DLochmelis33/UCheck/blob/master/CHANGELOG.md)
+[![MIT License Badge][license-badge]][license]
+[![Java CI with Gradle](https://img.shields.io/github/workflow/status/DLochmelis33/UCheck/Java%20CI%20with%20Gradle)](https://github.com/DLochmelis33/UCheck/actions/workflows/on_push.yml)
+[![contributions](https://img.shields.io/github/contributors/DLochmelis33/UCheck)](https://github.com/DLochmelis33/UCheck/graphs/contributors)
+![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)
+## Introduction
+UCheck is created to help people buy a variety of goods at the lowest prices. 
+
+### What can you do with UCheck?
+* Scan your check after shopping
+* Agregate all purchase information (prices, items, shop, etc) in UCheck database
+* Use the collected infornation to find necessary goods at the lowest prices
+
+For more information visit our [project wiki](https://github.com/DLochmelis33/UCheck/wiki)!
+
+## Installation
+
+1. Clone the repo: 
+```bash
+git clone git@github.com:DLochmelis33/UCheck.git
+```
+
+2. Execute Gradle
+
+## Authors
+- Gleb Solovev ([@GlebSolovev](https://github.com/GlebSolovev))
+- Denis Lochmelis ([@DLochmelis33](https://github.com/DLochmelis33))
+- Ekaterina Itsenko ([@Soykaa](https://github.com/Soykaa))
+
+## Acknowledgements
+We are very grateful to our wonderful teachers for their support and assistance during the whole project:
+- Vladislav Tankov
+- Timofey Bryksin
+
+## Contribute
+Contributions are welcome! Before creating a pull request, make sure that it adheres to the following guidelines:
+
+- Alphabetize your entry

```

Comment:
What does this mean?

Reply:
hm, nevermind, I think it's useless, my mistake